docs/scripts/getTypeScriptProjects.ts

```diff
@@ -0,0 +1,106 @@
+import path from 'path';
+import * as ts from 'typescript';
+import { Project, ProjectNames, Projects } from './api/utils';
+
+const workspaceRoot = path.resolve(__dirname, '../../');
+
+interface CreateProgramOptions {
+  name: ProjectNames;
+  rootPath: string;
+
+  /**
+   * Config to us to build this package
```

Comment:
```suggestion
   * Config to use to build this package.
```

All other descriptions are missing the period too.